e2e_correctness/test_modules.py

```diff
@@ -174,11 +190,56 @@ def _run_test(test_dir: str, memgraph_db: Memgraph, neo4j_driver: neo4j.BoltDriv
     ), "The graphs are not equal, check the logs for more details"
 
 
+def _run_path_test(
+    test_dir: str, memgraph_db: Memgraph, neo4j_driver: neo4j.BoltDriver
+):
+    """
+    Run input queries on Memgraph and Neo4j and compare path results after running test query
+    """
+    input_cyphers = (
+        test_dir.joinpath(TestConstants.INPUT_FILE).open("r").readlines()
+    )
+    logger.info(f"Importing data from {input_cyphers}")
+    mg_execute_cyphers(input_cyphers, memgraph_db)
+    logger.info("Imported data into Memgraph")
+    neo4j_execute_cyphers(input_cyphers, neo4j_driver)
+    logger.info("Imported data into Neo4j")
+
+    test_dict = _load_yaml(test_dir.joinpath(TestConstants.TEST_FILE))
+    logger.info(f"Test dict {test_dict}")
 
+    logger.info(
+        f"Running query against Memgraph: {test_dict[TestConstants.MEMGRAPH_QUERY]}"
+    )
+    memgraph_results = memgraph_db.execute_and_fetch(
+        test_dict[TestConstants.MEMGRAPH_QUERY]
+    )
+    memgraph_paths = parse_mem(memgraph_results)
+    logger.info("Done")
+
+    logger.info(
+        f"Running query against Neo4j: {test_dict[TestConstants.NEO4J_QUERY]}"
+    )
+    neo4j_results = execute_query_neo4j(
+        neo4j_driver, test_dict[TestConstants.NEO4J_QUERY]
+    )
+    neo4j_paths = parse_neo4j(neo4j_results)
+    logger.info("Done")
+
+    assert memgraph_paths == neo4j_paths
+
+def check_path_option(test_dir):
```

Comment:
@antoniofilipovic @imilinovic is this what you envisioned, this check if exists config and option